parcel/kyuubi-parcel-src/meta/alternatives.json

```diff
@@ -0,0 +1,8 @@
+{
+    "kbeeline": {
+      "destination": "/usr/bin/kbeeline",
```

Comment:
To distinguish w/ hive beeline, maybe we should rename our scripts to `kyuubi-beeline`?
I saw that aliyun did such thing https://help.aliyun.com/document_detail/440233.html

And a background, CDH does not ship STS, so there is no conflict between hive beeline and spark beeline

cc @turboFei @yaooqinn